prac_06/guitars.py

```diff
@@ -0,0 +1,34 @@
+"""
+Guitars
+Estimated: 35 minutes
+Actual: 28 minutes
+"""
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Add guitar objects to a list and print the attributes in a format."""
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        item = Guitar(name, year, cost)
+        guitars.append(item)
+        print(f"{item} added.")
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    print("These are my guitars:")
+    for i, guitar in enumerate(guitars, 1):
+        if guitar.is_vintage():
+            vintage_string = " (vintage)"
```

Comment:
Why is there a space at the start of this string when you're using an f string in the print statement? Maybe the sapce should be there? Could also be replaced with a turnary operator? (Not needed as the code works the same either way)

Reply:
Good point! Moved the space from the string to the print statement and changed the if-else lines to a single ternary operator.